### PR TITLE
feat: update shopify integration

### PR DESCRIPTION
**Pull request by oJshua**
- move React app to Parcel build 
- React app implements Figma mockups
- move reference API from NestJS to tsoa for simplicity
- Complete onboarding path
- Complete checkout path

### CODE REVIEW

**Comment by cogoo** [comment]
🥲 Sad to see NestJS go

File: shopify/api/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "main": "src/server.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsoa spec-and-routes && tsc --outDir build --experimentalDecorators",
+    "routes": "tsoa routes",
+    "start": "node build/src/server.js"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.5.6",
+    "@solana/pay": "^0.1.0-alpha.1",
```
[the remark this answers is not in the document]
Comment (oJshua) [reply shopify/api/package.json:13]: Thanks, will update! #yolo'd the merge, so next time I'll tag you as a reviewer.

**Comment by oJshua** [comment]
> smiling_face_with_tear Sad to see NestJS go

NestJS is great! Wanted to keep it simple for the reference app.

File: shopify/api/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jordansexton) [reply shopify/api/package.json:13]: All good, I actually made this comment last night and didn't submit the review 😅

**Pull request merged**